quiz-app/README.md

```diff
@@ -1,35 +1,118 @@
 # Quiz app
 
-This quiz app helps you display questions and answers in a quiz format. 
+These quizzes are the pre- and post-lecture quizzes for the data science curriculum at https://aka.ms/webdev-beginners
 
-## Install the app
+## Adding a translated quiz set
 
-Run `npm install` to install the dependencies.
+Add a quiz translation by creating matching quiz structures in the `assets/translations` folders. The canonical quizzes are in `assets/translations/en`. The quizzes are broken into several groupings. Make sure to align the numbering with the proper quiz section. There are 40 quizzes total in this curriculum, with the count starting at 0.
 
-```sh
-npm install
-```
+After editing the translations, edit the index.js file in the translation folder to import all the files following the conventions in `en`.
 
-## Build the app
+Edit the `index.js` file in `assets/translations` to import the new translated files.
 
-To build the solution, run `npm run build`.
+Then, edit the dropdown in `App.vue` in this app to add your language. Match the localized abbreviation to the folder name for your language.
 
-```sh
-npm run build
-```
+Finally, edit all the quiz links in the translated lessons, if they exist, to include this localization as a query parameter: `?loc=fr` for example.
 
-## Run
+## Run the Quiz App locally
 
-To run the solution, run `npm run dev`.
+### Prerequisites
 
-```bash
-npm run dev
-```
+- A GitHub account
+- [Node.js and Git](https://nodejs.org/)
+
+### Install & Setup
+
+1. Create a repository from this [template](https://github.com/new?template_name=Web-Dev-For-Beginners&template_owner=microsoft) 
+
+1. Clone your new repository, and navigate to the quiz-app
+
+   ```bash
+   git clone https://github.com/your-github-organization/repo-name
+   cd repo-name/quiz-app
+   ```
+
+1. Install the npm packages & dependencies
+
+   ```bash
+   npm install
+   ```
+
+### Build the app
+
+1. To build the solution, run:
+
+   ```bash
+   npm run build
+   ```
+
+### Start the App
 
-## Lint
+1. To run the solution, run:
 
-To ensure the code is linted, run `npm run lint`.
+    ```bash
+    npm run dev
+    ```
+
+### [Optional] Linting
+
+1. To ensure the code is linted, run:
+
+    ```bash
+    npm run lint
+    ```
+
+## Deploy the Quiz-app to Azure 
+
```

Comment:
To deploy your Quiz app, you will need to have an Azure account. Sign up for one for free [here](https://aka.ms/azure-free).